doc/languages-frameworks/python.section.md

```diff
@@ -297,13 +297,13 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   nativeBuildInputs = [
-    python3.pkgs.setuptools
-    python3.pkgs.wheel
+    python3Packages.setuptools
+    python3Packages.wheel
```

Comment:
Occurred to me after the fact, but the manual should probably alsoclarify that the interpreter should be accessed through python3Packages.python in this case, opposed to taking python3 as an argument that could diverge ftom python3Packages